### DISCUSSION TITLE
How can I refresh data

### DISCUSSION THREAD

**Discussion by danillo-hilbs** [nuxt/nuxt#17255] [Questions] [answered]
How can i refresh() data? data on screen not is updated

![image](https://user-images.githubusercontent.com/34604307/179804741-927c5285-7e22-47be-8b03-fe9a3ec12cb1.png)

**Comment by Flanker72** [comment, marked as answer]
You should return `dimensions` as `ref` from your `useGetDimensions` to trigger the reactivity.
Now `refresh` is called, new dimensions appeared in the `useGetDimensions`, however your `reactiveDimensions` still stay the same.
